src/Fixer/ControlStructure/SimplifiedIfReturnFixer.php

```diff
@@ -93,26 +90,29 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
     {
         for ($ifIndex = $tokens->count() - 1; 0 <= $ifIndex; --$ifIndex) {
             $ifToken = $tokens[$ifIndex];
+
             if (!$ifToken->isGivenKind([T_IF, T_ELSEIF])) {
                 continue;
             }
 
             $startParenthesisIndex = $tokens->getNextTokenOfKind($ifIndex, ['(']);
             $endParenthesisIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $startParenthesisIndex);
+            $firstCandidateIndex = $tokens->getNextMeaningfulToken($endParenthesisIndex);
```

Comment:
It seems to me that this addition slows performance down with no added benefit: what am I missing?

Reply:
Please explain why you search for '(', than go back one (up to 4 times) and than search for it again.
That is;
- search for '('; `$startParenthesisIndex = $tokens->getNextTokenOfKind($ifIndex, ['(']);` line 98
- than go back one (up to 4 times); `$sequenceFound = $tokens->findSequence($sequenceSpec['sequence'], $endParenthesisIndex - 1);` line 104

forces you to refind something you already know (the position of `)`) up to four times, and you need to declare it as part of the sequence 4 times/
In my diff you see the removal of;
- going back one (up to 4 times)
- search for the ')' up to for times, because it is not needed at all